### ISSUE TITLE
Custom CSS does not work when appearence disabled [BUG] 

### ISSUE THREAD

**Issue by ayyilmaz** [AmruthPillai/Reactive-Resume#1233]
**Describe the bug**

**Product Flavor**

- [ ] Managed (https://rxresu.me)

**To Reproduce**

1. Go to 'rxresu.me'
2. Click on 'cv'
3. Scroll down to 'custom css section'
4. write your own commands and it does not effect, you need to enable apeearence of css but this broke CV. Apeears millions rectangle. If you disable it by click to eye icon css command becomes passive. 

I need to change this command for Glalie template because there is huge blank area at summary section as default. I would like to make its margin values as 1rem

Glalie summary section's default `<hr>` tag css
```
.markdown :where(hr):not(:where([class~=not-prose] *)) {
    margin-top: 2.8571429em;
    margin-bottom: 2.8571429em
}
```

**Expected behavior**

CSS command should effect whenever eye icon turn off or turn on. 

**Screenshots**

![image](https://user-images.githubusercontent.com/24845098/219885074-201378ae-bb15-46c8-af3e-0456cf20a8c3.png)

**Desktop (please complete the following information):**

- OS: [Windows 10] 
- Browser : [chrome ] 
- Version : [110.0.5481.104]

**Comment by owenhenley** [comment]
When using the CSS, delete the `* {` block of code

**Comment by astaticflow** [comment]
I do not agree that this is a bug. The rectangles that appear is the 1px outline referenced in the custom CSS panel. If you comment out this section or delete it, you won't see them. The custom CSS is being applied appropriately.
